recipe/meta.yaml

```diff
@@ -25,7 +25,8 @@ requirements:
     - pip
   run:
     - python >={{ python_min }}
-    - sympy
+    - sympy >=1.12
+    - lxml >=5.3
```

Comment:
@pelson I'm using Python 3.13 min deps here to keep it noarch. Thanks to CFEP-25 we ensure it builds and runs fine on Python <3.13 too.